package/gluon-dns-cache/check_site.lua

```diff
@@ -0,0 +1,3 @@
+need_string_match('dns.cacheentries', '^%d$')
```

Comment:
This is plain wrong, the value should be a number (as you explain in the README).